lib/browser.js

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+import historyPolyfill from 'html5-history-api';
```

Comment:
hmm, not sure i follow about the global namespace implementation?  not sure how that would make the build smaller if were still requiring it in?

And i just assumed since this library supports legacy browsers that new APIs should do the same.  I think it would be inconsistent behavior if some APIs supported legacy browsers while others did not.

eg. setRoute vs. replaceRoute. setRoute supports older versions of IE but without the history polyfill, replaceRoute would only support up to IE 10 or 11 I think.

What do you think?